.changeset/long-rats-share.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/ember-flight-icons": minor
+---
+
+`FlightIcon` component - Convert to typescript & publish types
```

Comment:
nit (we have some very specific guidance about [consistent terminology](https://github.com/hashicorp/design-system/blob/main/wiki/Website-Changelog.md#consistent-terminology) for the changelog)
```suggestion
`FlightIcon` - converted component to TypeScript and published types
```